[Jormun - NAV-3444] fix multiple first and last sections

princefr opened the pull request.

princefr committed:
add penality bike parking

kadhikari committed:
[jormun]: Fix request_datetime in ridesharing

kadhikari committed:
Formatting 1

princefr committed:
change way of doing things

princefr committed:
revert to default changes

princefr committed:
add tests cases

princefr committed:
revert some changes

princefr committed:
pre commit fix

princefr committed:
adding bragi call

kadhikari committed:
jormungandr: Fix geographic position address

kadhikari committed:
Formatting and adding comment

kadhikari committed:
Another formatting

kadhikari committed:
Comments updated

princefr committed:
adding walking time

princefr committed:
update navitia-proto version

princefr committed:
add new proto version

princefr committed:
Add navitia-proto submodule

princefr committed:
restore gitmodule

princefr committed:
restor routing

princefr committed:
fix after reviews

princefr committed:
remove unacessary code for park_mode

princefr committed:
reverting some code review changes that causes bugs

princefr committed:
adding integration tests

princefr committed:
update park mode help reference

princefr committed:
adding server_default

princefr committed:
update server default value

princefr committed:
fix multiple first and last sections

princefr committed:
add tests and fix vias

princefr committed:
remove distance comment

princefr committed:
add tests

azime commented on source/jormungandr/tests/routing_tests_experimental.py:
```diff
@@ -1563,3 +1563,101 @@ def test_bike_with_parking_penalty_first_section_and_access_points(self):
+        journeys = get_not_null(response, 'journeys')
+        assert len(journeys) > 0
+        for journey in journeys:
+            if len(journey['sections']) > 1:
```
https://github.com/hove-io/navitia/pull/4320#discussion_r1867315854

princefr committed:
fix times and tests

princefr committed:
max_bike_duration_to_pt fix and arrival time fix

princefr committed:
max_bike duration

kadhikari committed:
jormungandr: Update poi objects in journeys if active by configuration

kadhikari committed:
Formatting

princefr committed:
fix different place type

princefr committed:
update protos

princefr committed:
fix instructions without adding parking instructions

princefr committed:
change parameter from array to string

princefr committed:
fix signe for arrival and max_bike_duration_to_pt

Commit:
add new backends_status

regroup

add request id

simplify response

update interface

add more infor

add test

princefr committed:
remove params and update helpers methods

princefr committed:
fixing departure times

princefr committed:
fix sign

azime commented on source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py:
```diff
@@ -391,6 +422,246 @@ def add_poi_access_point_in_sections(fallback_type, via_poi_access, sections):
+    journey.nb_sections += 2
+
+
+def _get_coords_from_pt_object(pt_object):
```
https://github.com/hove-io/navitia/blob/dev/source/jormungandr/jormungandr/utils.py#L546

princefr committed:
fix double function
